assets/js/composer.js

```diff
@@ -404,6 +430,19 @@ PageComposer.prototype = {
         headers: {
           Accept: 'application/json',
         },
+        error(resp) {
```

Comment:
There are more functions that need this kind of modifications. 

What happens here is:
1. We were using success method for both success and error (take a look at the if / else)
2. We were relying on a "bug" on admin-bundle to print directly the output as html on the page when an error happens
3. for new code like `handleBlockErrors`, we should try to use as less as jQuery as possible, there are good alternative for most of the usages.

So I would suggest:

1. Take a look at my PR for SonataAdminBundle, since it will change how you recover errors from the ajax call
2. You will need to probably cleanup logic on the ELSE part of the success method
3. There could be more cases where the if/else doesn't make sense and we also need a error method
4. Try to reduce the amount of jQuery if you can on the new methods you create (don't try to refactor the whole thing, because it will be harder to review)

Reply:
Thanks @jordisala1991 I'll do that 👍